pallets/spaces/src/types.rs

```diff
@@ -18,8 +18,8 @@ pub struct Space<T: Config> {
     /// Unique sequential identifier of a space. Examples of space ids: `1`, `2`, `3`, and so on.
     pub id: SpaceId,
 
-    pub(super) created: WhoAndWhenOf<T>,
-    pub(super) updated: Option<WhoAndWhenOf<T>>,
+    pub created: WhoAndWhenOf<T>,
+    pub updated: Option<WhoAndWhenOf<T>>,
```

Comment:
Why?

This question applies to all changes to the file.

Reply:
We need access to them in tests.